components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -463,4 +464,25 @@ - (void)testBottomDrawerCornersAPIFullScreen {
   XCTAssertEqual(self.drawerViewController.maskLayer.minimumCornerRadius, 3.f);
 }
 
+- (void)testBottomDrawerHandle {
+  // When
+  [self.presentationController presentationTransitionWillBegin];
+  // Then
+  XCTAssertNotNil(self.presentationController.topHandle);
+  XCTAssertEqual(CGRectGetWidth(self.presentationController.topHandle.frame), 24.f);
+  XCTAssertEqual(CGRectGetHeight(self.presentationController.topHandle.frame), 2.f);
+  XCTAssertEqual(self.presentationController.topHandle.layer.cornerRadius, 1.f);
+  XCTAssertEqual(self.presentationController.topHandle.hidden, YES);
+}
```

Comment:
Can we add a test that verifies the alpha behavior of the handle as it scrolls to the top? If not, please file an issue to explore making it possible to do so.

Reply:
created a follow up bug: https://github.com/material-components/material-components-ios/issues/5636